### ISSUE TITLE
[Feature Request] Show anki tags and/or flags on words that already have cards created

### ISSUE THREAD

**Issue by soriac** [FooSoft/yomichan#1626]
When reviewing cards I often notice that the context I added is bad, insufficient, or maybe I even forgot to add the actual context. I like to tag and flag these words so that whenever I want to clean my decks up I can quickly find "bad cards" and deal with them.

Sometimes, when reading japanese text I'll see one of the words I have flagged and remember that it's flagged, so I can delete the old card or update its context. However most of the time I simply come across these words, see that they already have a card and then not fix it because I don't know that the card was bad.

It would be nice to be able to show all tags and/or flags for any words that have an anki card associated with them, so that I can quickly see "oh, this is a red card tagged with `insufficient_context`, let's fix it" and not have to be on a constant lookout while reading.

**Comment by Thermospore** [comment]
Maybe not what you are looking for but personally I would
 -[make cards out of sentences instead of words](https://refold.la/roadmap/stage-2/a/basic-sentence-mining#Mine-Sentences-Not-Words)
 -add a screenshot or photo of whatever you were consuming for further context
 
This should prevent the issue from the root
 
Here are what my sentence cards look like for example

front:
![image](https://user-images.githubusercontent.com/35350819/116037097-17930580-a614-11eb-9be1-81f209b20003.png)

back (usually I only learn one word per card but these were easy):
![image](https://user-images.githubusercontent.com/35350819/116037392-7eb0ba00-a614-11eb-93b9-3b0a97b93a0e.png)

**Comment by soriac** [comment]
Yes! I actually make Audio Cards, with sentences, pictures and audio, mining from visual novels. 

The problem is sometimes, in the future, I realize the card I made wasn't that good, or I misunderstood the word or context. For example, I created a card for 融通 where the sentence actually uses 融通が利く.

Sometimes while reading I'll be too focused and just hit the plus button but then forget to add the rest of the context.

Sometimes there's a word I really want to add, because it comes up over and over and I need some help remembering the reading, but I just can't find a good context for within the novel itself.

And finally, sometimes the context is NSFW and while I'm fine with that, I'd rather keep my decks as SFW as possible.

In those cases, I'd rather not have perfect be the enemy of good and just make a "meh" card, that I can maybe fix later. Right now I just tag and flag those cards, and whenever I have some free time try to either delete or unflag all flagged cards (by finding some other context for them).

I thought of moving those cards to a separate deck, but then the new card order gets messed up and they either always go to the very end or the very start of the queue.

With this feature, my workflow could be something like: 
1. See a word i'm unsure of, select it
2. See that I have a flagged/tagged card created
3. Go look at the card, fix the context
4. Become 上手

I had some trouble setting up the dev environment on Windows but I just managed to successfully install the dependencies for node-gyp via chocolatey, so I'm going to throw a few hours at this and hopefully submit a PR soon, if you guys think this feature could be useful (if not i'll just have to maintain my own fork 😂)
